src/resizer/resizer.ts

```diff
@@ -1,6 +1,6 @@
 /*
 Copyright 2018 New Vector Ltd
-Copyright 2019 The Matrix.org Foundation C.I.C.
+Copyright 2019, 2020 The Matrix.org Foundation C.I.C.
 
```

Comment:
while you're here, please collapse New Vector Ltd into the foundation (we've done this before, and should have done it a long time ago)